application/frontend/controllers/AuthController.php

```diff
@@ -231,12 +244,28 @@ protected function getReturnTo()
         return $returnTo;
     }
 
+    /**
+     * Get a return-to url for where to send browser in the event of an error
+     * If it's a relative url (starting with '/') it will be prefixed with uiUrl
+     * @return array|mixed|string
```

Comment:
Wouldn't this always return a string?

Reply:
Yep. Looks like this was auto-generated.